examples/inference-deployments/mpt/mpt_7b_ft_handler.py

```diff
@@ -171,6 +178,8 @@ def __init__(self,
 
         self.end_id = end_id
 
+        comm.initialize_model_parallel(tensor_para_size, pipeline_para_size)
```

Comment:
do we need this?

Reply:
we call `dist.init_process_group(backend=dist.Backend.MPI)` in the launcher tho -- won't this conflict?